src/nxdoc/web-ui/nuxeo-elements/nuxeo-elements-tutorial.md

```diff
@@ -29,7 +29,7 @@ Watch the related courses on Nuxeo University:</br>
 
 ## Requirements
 
-- **[Node.js](https://nodejs.org/)** is a JavaScript runtime built on Chrome's V8 JavaScript engine. Almost every tool out there for client-side development is built with Node.js and distributed with npm, the package manager for Node.js. Make sure you download and install your OS-specific version first (last verified versions for this tutorial are node v10.12.0 and npm 6.4.1)
+- **[Node.js](https://nodejs.org/)** is a JavaScript runtime built on Chrome's V8 JavaScript engine. Almost every tool out there for client-side development is built with Node.js and distributed with npm, the package manager for Node.js. Make sure you download and install your OS-specific version first (last verified versions for this tutorial are node v10.x)
```

Comment:
Should we really specify the Node version here?

Reply:
I mean, that it is v10.23.0 now? Or should you keep v10.x?

Reply:
TBH, I think it would be better to have it in a single centralized (publicly accessible) place, per LTS version, just to avoid having to go around replacing things everywhere.
Maybe in the README files of nuxeo-elements and nuxeo-web-ui? Or maybe just in nuxeo-elements and we can point the nuxeo-web-ui section to the nuxeo-elements one? If necessary, we could reference those README sections here or in the wiki too.

Reply:
Because Elements can be used without Web UI andm in theory, the former can require a lower version of node than the latter, I think we should document the version for both elements and Web UI.

Maybe we could document it [here for Elements](https://doc.nuxeo.com/nxdoc/nuxeo-elements/), and [here for Web UI](https://doc.nuxeo.com/nxdoc/web-ui-overview/), wdyt?

Reply:
I think it would make a bit more sense to have those technical details in the repos themselves, rather than in doc website, but I'm fine with either.